### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by dtolnay** [phaazon/proc-macro-faithful-display#2]
I haven't quite put together what this crate is doing. Could you provide an example of displaying a TokenStream in the default way that is not faithful, together with what the faithful output of this crate would be for the same TokenStream?

**Comment by phaazon** [comment]
Yeah, I could move documentation from my blog post to this crate doc. See [the blog post here](https://phaazon.net/blog/proc-macro-faithful-display).
